### PR TITLE
README.md spec files are only actual for the stuff buitl from source

### PR DESCRIPTION
**Pull request by techtonik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,3 +78,9 @@ The toolchains are fully portable and can be unpacked without admin rights or ot
 build systems doesn't like paths with spaces or other special characters it is recommended to avoid such paths. 
 To use the toolchain it is necessary to find the executables in the `bin` folder. This can be achieved with 
 extending the `PATH` environment variable 
+
+#### Configuring toolchain for CPython version
```
Comment (ogrisel) [inline README.md]: => Configuring the toolchain for a specific version of CPython

Comment (techtonik) [inline README.md]: Fixed.

**Comment by ogrisel** [comment]
Thanks!

**Comment by techtonik** [comment]
No problem.

**Pull request merged**